src/racer/project_model.rs

```diff
@@ -0,0 +1,216 @@
+use std::path::{Path, PathBuf};
+
+pub trait RacerProjectModelProvider {
```

Comment:
Do we really need `Racer` prefix here?
From other crates it looks `racer::RacerProjectModelProvider`